scripts/zones/Batallia_Downs_[S]/npcs/Thorben.lua

```diff
@@ -0,0 +1,65 @@
+-----------------------------------
+-- Area: Batallia Downs (S)
+--  NPC: Thorben
+-- Type: Quest NPC
+-- !pos  175.346,8.038,-419.244 84
+-----------------------------------
+require("scripts/globals/keyitems")
+require("scripts/globals/settings")
+require("scripts/globals/quests")
+require("scripts/globals/npc_util")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+    local lostInTranslocation = player:getQuestStatus(CRYSTAL_WAR, dsp.quest.id.crystalWar.LOST_IN_TRANSLOCATION)
+    local leftMapPiece = player:hasKeyItem(dsp.ki.LEFT_MAP_PIECE)
+    local middleMapPiece = player:hasKeyItem(dsp.ki.MIDDLE_MAP_PIECE)
+    local rightMapPiece = player:hasKeyItem(dsp.ki.RIGHT_MAP_PIECE)
```

Comment:
Can pop a `hasAnyPiece` here which is your `or`s, then below check for:
`var == 1 and hasAnyPiece`
`ACCEPTED and hasAnyPiece`
`ACCEPTED and not hasAnyPiece`

Reply:
Awesome! thank you for pointing that out 😸️ just fixed it this way to make it easy to read.